cli/tink/cmd/workflow/create.go

```diff
@@ -40,14 +35,14 @@ var createCmd = &cobra.Command{
 func addFlags() {
 	flags := createCmd.PersistentFlags()
 	flags.StringVarP(&template, "template", "t", "", "workflow template")
-	flags.StringVarP(&target, "target", "r", "", "workflow target")
+	flags.StringVarP(&hardware, "hardware", "r", "", "workflow target hardwares")
```

Comment:
Replace `r` with `h`.

Reply:
The -h flag is already reserve for `help` I guess because initally I used -h flag but it gave me the error of duplicate flags.